include/functions.h

```diff
@@ -1022,14 +1024,14 @@ u32 PreNmiBuff_IsResetting(PreNmiBuff* this);
 void MsgEvent_SendNullTask();
 f32 OLib_Vec3fDist(Vec3f*, Vec3f*);
 // ? OLib_Vec3fDistXZ(?);
-// ? func_8007C058(?);
-// ? func_8007C0A8(?);
+// ? OLib_ClampMinDist(?);
+// ? OLib_ClampMaxDist(?);
 // ? OLib_Vec3fDistNormalize(?);
-// ? OLib_VecSphRot90ToVec3f(?);
+// ? OLib_VecSphGeoToVec3f(?);
 // ? OLib_Vec3fToVecSph(?);
-VecSph* OLib_Vec3fToVecSphRot90(VecSph* arg0, Vec3f* arg1);
-VecSph* OLib_Vec3fDiffToVecSphRot90(VecSph* arg0, Vec3f* a, Vec3f* b);
-// ? func_8007C4E0(?);
+VecSph* OLib_Vec3fToVecSphGeo(VecSph* arg0, Vec3f* arg1);
+VecSph* OLib_Vec3fDiffToVecSphGeo(VecSph* arg0, Vec3f* a, Vec3f* b);
+// ? OLib_Vec3fDiffRad(?);
 // ? func_8007C680(?);
```

Comment:
i imagine a bunch of these should be added in here, it seems like stuff that should be publicly exposed